magda-gateway/package.json

```diff
@@ -43,6 +43,7 @@
     "passport-facebook": "^2.0.0",
     "passport-google-oauth20": "^1.0.0",
     "passport-local": "^1.0.0",
+    "passport-wsfed-saml2": "^3.0.17",
```

Comment:
I think we might need to fork our version of this with the x509 and cryptiles problems patched out :(. No need for you to get to this before you go though.

Reply:
https://github.com/auth0/passport-wsfed-saml2/pull/129

Reply:
Will native x509 dependency cause problems? It just needs build tools for npm install.

Reply:
Apparently it breaks on node > 8?

Reply:
There's already a PR to get rid of it, but it's been sitting since October :(